cmd/example/main.go

```diff
@@ -65,11 +66,19 @@ func getProvider(ctx context.Context, key, mode string) (client.Provider, error)
 	switch mode {
 	case "api":
 		provider, err = client.ConnectAPIProvider(ctx, key, rk)
-	case "in-memory":
-		provider, err = client.BackendInMemoryProvider(ctx, &client.InMemoryProviderOptions{
-			APIKey:         key,
-			RepositoryKey:  *rk,
-			UpdateInterval: 10 * time.Second,
+	case "backend":
```

Comment:
nit: use constants?

Reply:
im gonna leave it as is since its an example script